conf/jasminegraph-server.properties

```diff
@@ -56,6 +56,8 @@ org.jasminegraph.graphsage=/var/tmp/jasminegraph/GraphSAGE/graphsage/
 
 #Scheduler Specific Parameters
 org.jasminegraph.scheduler.enabled=true
+#PerformanceCollector Scheduler Timing. Run once every 120 seconds
+org.jasminegraph.scheduler.performanceCollector.timing=2
```

Comment:
Need to change "performanceCollector" to "performancecollector". Because all the characters of property variable definitions need to be lower-case.

Reply:
Lets change the org.jasminegraph.scheduler.performancecollector.timing to specify values in seconds.
e.g.,

org.jasminegraph.scheduler.performancecollector.timing=120